sympy/polys/matrices/domainmatrix.py

```diff
@@ -21,6 +21,171 @@
 
 
 class DomainMatrix:
+    r"""
+
+        Module for the DomainMatrix class.
```

Comment:
Since this is a docstring for the class it should describe the class rather than the module